conda/recipes/librmm/recipe.yaml

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  GIT_DESCRIBE_HASH: ${{ env.get("GIT_DESCRIBE_HASH") }}
+  GIT_DESCRIBE_NUMBER: ${{ env.get("GIT_DESCRIBE_NUMBER") }}
+
+recipe:
+  name: librmm-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - cmake ${{ cmake_version }}
+      - ninja
+      - ${{ compiler('c') }}
+      - ${{ compiler('cxx') }}
+      - ${{ compiler('cuda') }}
+      - cuda-version =${{ cuda_version }}
+      - ${{ stdlib("c") }}
+
+  build:
+    script:
+      file: build.sh
+      secrets:
+        - AWS_ACCESS_KEY_ID
+        - AWS_SECRET_ACCESS_KEY
+        - AWS_SESSION_TOKEN
+
+      env:
+        CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR", default="Ninja") }}
+        PARALLEL_LEVEL: ${{ env.get("PARALLEL_LEVEL", default="4") }}
+        SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET", default="") }}
+        SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT", default="") }}
+        SCCACHE_REGION: ${{ env.get("SCCACHE_REGION", default="") }}
+        SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL", default="") }}
+        SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS", default="") }}
+        SCCACHE_S3_KEY_PREFIX: librmm-${{ env.get("RUNNER_ARCH", default="X64") | replace("X64", "linux64") | replace("ARM64", "aarch64") | lower }}
+
+outputs:
+  - package:
+      name: librmm
+      version: ${{ version }}
+    build:
+      number: ${{ GIT_DESCRIBE_NUMBER }}
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ GIT_DESCRIBE_HASH }}_${{ GIT_DESCRIBE_NUMBER }}
+      script: install_librmm.sh
+    requirements:
+      host:
+        - cmake ${{ cmake_version }}
+        - cuda-version =${{ cuda_version }}
+        - fmt ${{ fmt_version }}
+        - spdlog ${{ spdlog_version }}
+      run:
+        - ${{ pin_compatible('cuda-version', upper_bound='x', lower_bound='x') }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - fmt ${{ fmt_version }}
+        - spdlog ${{ spdlog_version }}
+      run_exports:
+        - ${{ pin_subpackage("librmm", upper_bound="x.x") }}
+      ignore_run_exports:
+        from_package:
+          - ${{ compiler('cuda') }}
+    tests:
+      - script:
+          - "test -d \"${PREFIX}/include/rmm\""
+    about:
```

Comment:
It's somewhat out of scope for this PR, but also not really since one reason I never considered this sooner was because I wasn't sure what rattler would support by the time that we got here: should we try using `load_from_file` to get this information from the pyproject.toml file now? [rattler-build has this functionality now](https://rattler.build/latest/experimental_features/#load_from_filefile_path). It's pretty low-priority but given all of the other cleanup that we're doing in this PR it might be nice to really single-source .